script.js

```diff
@@ -0,0 +1,219 @@
+
+class Player {
+	constructor(name, symbol) {
+		this.name = name,
+		this.symbol = symbol,
+		this.moves = []
+	}
+
+	checkWin (board) {
```

Comment:
@YellowPipe and @zinagithub, 

After sharing this last review:
```javascript
const checkWinner = (player) => {
    return winCombinations.some(combo => combo.every(element =>   player.moves.includes(element)));
  }
```
we still built a very complex `checkWin` method here. I already demonstrated the use of `every` and `some` yet we still used a nested `for` loop here.